computers/daint-hybrid/cp2k-6.1@daint-hybrid.yml

```diff
@@ -1,11 +1,12 @@
-label: "cp2k-6.1-18464"
-description: "CP2K with CUDA on daint, compiled by CSCS"
-input_plugin: "cp2k"
+---
+label: cp2k-6.1-18464
+description: CP2K with CUDA on daint, compiled by CSCS
+input_plugin: cp2k
 on_computer: true
-remote_abs_path: "/apps/daint/UES/jenkins/7.0.UP01/gpu/easybuild/software/CP2K/6.1-CrayGNU-19.10-cuda-10.1/bin/cp2k.psmp"
-computer: "daint-hybrid"
+remote_abs_path: /apps/daint/UES/jenkins/7.0.UP01/gpu/easybuild/software/CP2K/6.1-CrayGNU-19.10-cuda-10.1/bin/cp2k.psmp
+computer: daint-hybrid
 prepend_text: |
-  ###  code prepend_text start ###
-  module load CP2K
-  ###  code prepend_text end ###
-append_text: " "
+    ###  code prepend_text start ###
+    module load CP2K
+    ###  code prepend_text end ###
+append_text: ' '
```

Comment:
I believe this can be an empty string. No need to keep a space here, right? Same for all the cases below.

Reply:
I think so, don't know why use space here originally. I will change this to empty string.

Reply:
Yes, this is something that will need to be fixed in AiiDA at some point. It is necessary.

Reply:
wow, didn't know that. Thanks!

Reply:
> Yes, this is something that will need to be fixed in AiiDA at some point. It is necessary at the moment.

Is there an issue for that?

Reply:
I checked and didn't find one for this issue specifically. You're welcome to create it

Reply:
https://github.com/aiidateam/aiida-core/issues/4300

Reply:
I think the issue has been resolved in the latest release of AiiDA.